src/h5web/vis-packs/core/heatmap/ScalingMesh.tsx

```diff
@@ -0,0 +1,34 @@
+import { useCanvasScales } from '../hooks';
+import Mesh, { MeshProps } from './Mesh';
+
+interface Props extends MeshProps {
+  minAbscissa: number;
+  maxAbscissa: number;
+  minOrdinate: number;
+  maxOrdinate: number;
+}
+
+function ScalingMesh(props: Props) {
+  const { minAbscissa, maxAbscissa, minOrdinate, maxOrdinate, ...meshProps } =
+    props;
```

Comment:
`ScalingMesh` only needs the limit coordinates and forward the rest of the props to `Mesh`. 

We could also make use of WebGL to scale a new `mesh` and have `Mesh` a children of this new `mesh` so that it is relatively positioned.